crates/uv/tests/pip_compile.rs

```diff
@@ -4668,7 +4668,7 @@ fn offline_registry() -> Result<()> {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because black==23.10.1 was not found in the cache and you require black==23.10.1, we can conclude that the requirements are unsatisfiable.
+      ╰─▶ Because black was not found in the cache and you require black==23.10.1, we can conclude that the requirements are unsatisfiable.
```

Comment:
The message improves here because it's the versions of black, not that specific version, that is missing in the cache